tests/components/mqtt/test_discovery.py

```diff
@@ -178,31 +176,6 @@ async def test_invalid_config(
     assert "Error 'expected int for dictionary value @ data['qos']'" in caplog.text
 
 
-async def test_only_valid_components(
```

Comment:
We could keep this test, but just remove the `assert f"Integration {invalid_component} is not supported" in caplog.text`

Reply:
done